website/content/docs/secrets/mongodbatlas.mdx

```diff
@@ -16,6 +16,8 @@ project or organization with appropriate role(s). The MongoDB Atlas Programmatic
 Private Keys are returned to the caller. To learn more about Programmatic API Keys visit the
 [Programmatic API Keys Doc](https://docs.atlas.mongodb.com/reference/api-docs/apiKeys/).
 
+Please do not confuse this with MongoDB Atlas Database secrets engine described [here](/vault/docs/secrets/databases/mongodbatlas).
```

Comment:
The thing you are linking to here is not a secrets engine - it is a database plugin

```suggestion
Please do not confuse this with MongoDB Atlas database plugin described [here](/vault/docs/secrets/databases/mongodbatlas).
```